### ISSUE TITLE
[E2E] Timm fbnetv3_b amp_bf16 training accuracy failed

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#707]
### 🐛 Describe the bug

Model list:
- [ ] `fbnetv3_b`

E0804 06:29:58.153000 934284 torch/_dynamo/utils.py:1558] RMSE (res-fp64): 0.30015, (ref-fp64): 0.05598 and shape=torch.Size([360]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.040000
E0804 06:29:58.154000 934284 torch/_dynamo/utils.py:1450] Accuracy failed for key name blocks.4.3.bn1.running_var
fail_accuracy

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/1d70431c072db889d9a47ea4956049fe340a426d
pytorch: d224857b3af5c9d5a3c7a48401475c09d90db296
device: pvc 1100, bundle: 0.5.3, driver: 803.61

**Comment by retonym** [comment]
low priority for ampbf16 training not included in Meta PyTorch dashboard

**Comment by mengfei25** [comment]
A100 is also failed

**Comment by retonym** [comment]
close the issue, due to A100 also fails
